pkg/build/build.go

```diff
@@ -23,19 +24,32 @@ func New(ctx *image.Context) *Builder {
 }
 
 func (b *Builder) Build() error {
+	log.Audit("Generating image customization components...")
+
 	if err := b.configureCombustion(b.context); err != nil {
 		return fmt.Errorf("configuring combustion: %w", err)
 	}
 
 	switch b.context.ImageDefinition.Image.ImageType {
 	case image.TypeISO:
-		return b.buildIsoImage()
+		log.Audit("Building ISO image...")
+		if err := b.buildIsoImage(); err != nil {
+			log.Audit("Error building ISO image")
```

Comment:
Just throwing out an idea for this one, let me know if you disagree. The message can be more descriptive e.g. "Building ISO image failed. Check build log file for details"

Reply:
This was actually my original intention but I completely forgot when it came to do it. Good call.